stats/stats-server/src/update_service.rs

```diff
@@ -107,12 +104,11 @@ impl UpdateService {
             force_update = force_full,
             "updating group of charts"
         );
-        let Ok(active_migrations) =
-            BlockscoutMigrations::query_from_db(self.blockscout.connection.as_ref())
-                .await
-                .inspect_err(|err| {
-                    tracing::error!("error during blockscout migrations detection: {:?}", err)
-                })
+        let Ok(active_migrations) = BlockscoutMigrations::query_from_db(&self.blockscout)
+            .await
+            .inspect_err(|err| {
+                tracing::error!("error during blockscout migrations detection: {:?}", err)
+            })
```

Comment:
we didn't encounter lack of retry to be a problem so far + failure here means that update will most definitely fail

(because there's also `migrations_table_exists_and_available` check within `query_from_db` that handles lack of table)